style.css

```diff
@@ -0,0 +1,105 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+}
+
+header {
+  min-height: 100vh;
+  width: 100%;
+  background-image: linear-gradient(rgba(1, 9, 30, 0.7), rgba(4, 9, 30, 0.7)),
+    url(./images/banner.png);
+  background-position: center;
+  background-size: cover;
+  position: relative;
+}
+
+.menu {
+  display: flex;
+  padding: 2% 6%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 150px;
+}
+.logo a {
+  text-decoration: none;
+}
+.menu-links {
+  flex: 1;
+  text-align: right;
+}
+
+.menu-links ul li {
+  list-style: none;
+  display: inline-block;
+  padding: 8px 12px;
+  position: relative;
+}
+
+.menu-links ul li a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.menu-links ul li::after {
+  content: "";
+  width: 0%;
+  height: 2px;
+  background: #551a8b;
+  display: block;
+  margin: auto;
+  /* transition: 0.5s; */
+}
+
+.menu-links ul li:hover::after {
+  width: 100%;
+  transition: 0.5s;
+}
+
+.hero {
+  width: 90%;
+  color: #fff;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Good job for managing to find a way of centering your hero text by using CSS property. But I think it would be better if your wrap the h1 and p elements in a container and then use flexbox to properly position them in the center as you wanted.

Reply:
[COMPLIMENT] Great job in using a polite tone 👍 💯